serverless/src/libs/dynamo/ethereum.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  PutCommand,
+  GetCommand,
+} from '@aws-sdk/lib-dynamodb';
+import Database from './database';
+
+class EthereumDatabase extends Database {
+  async storeTransaction(txHash: string): Promise<void> {
```

Comment:
```suggestion
  async storeTransaction(txHash: string): Promise<PutCommandOutput> {
```

just a suggestion, this might throw a warning